lib/dotcom_web/live/schedule_finder_live.ex

```diff
@@ -119,4 +248,168 @@ defmodule DotcomWeb.ScheduleFinderLive do
+    ~H"""
+    <time datetime={@time} class="tabular-nums whitespace-nowrap">
+      {format!(@time, :hour_12_minutes)}
+    </time>
+    """
+  end
+
+  attr :route, Route, required: true
+  attr :departures, :list, required: true
+  attr :loaded_trips, :map, required: true
+
+  defp departures_table(assigns) do
+    ~H"""
+    <div class="grid grid-cols-1 divide-y-[1px] divide-gray-lightest border-[1px] border-gray-lightest">
+      <.unstyled_accordion
+        :for={departure <- @departures}
+        id={departure.schedule_id}
+        summary_class="flex items-center gap-sm hover:bg-brand-primary-lightest p-sm"
+        phx-hook="SFTripRow"
+        data-trip={departure.trip_id}
+        data-stop_sequence={departure.stop_sequence}
+      >
+        <:heading>
+          <div class="flex items-center gap-sm w-full">
+            <RouteComponents.route_icon route={@route} />
+            <div>
+              {departure.headsign}
+              <div :if={@route.type == 2 && departure.trip_name} class="text-sm">
+                {~t(Train)} {departure.trip_name}
+              </div>
+            </div>
+          </div>
+          <.formatted_time time={departure.time} />
+        </:heading>
+        <:content>
+          <div class="divide-y-[1px] divide-gray-lightest border-t border-gray-lightest [&>*:first-child_.top]:invisible [&>*:last-child_.bottom]:invisible">
+            <.async_result
+              :let={arrivals}
+              assign={Map.get(@loaded_trips, departure.schedule_id, AsyncResult.loading())}
+            >
+              <:loading>
+                <div class="p-lg text-gray">Loading arrivals...</div>
+              </:loading>
+              <:failed :let={fail}>
+                <.error_container title={inspect(fail)}>
+                  {~t"There was a problem loading arrivals"}
+                </.error_container>
+              </:failed>
+              <.arrival_row
+                :for={arrival <- arrivals}
+                :if={arrivals}
+                route={@route}
+                arrival={arrival}
+              />
+            </.async_result>
+          </div>
+        </:content>
+      </.unstyled_accordion>
+    </div>
+    """
+  end
+
+  attr :arrival, FutureArrival, required: true
+  attr :route, Route, required: true
+
+  defp arrival_row(assigns) do
+    ~H"""
+    <div class="px-[1.75rem] py-sm gap-md flex justify-between items-center">
+      <div
+        class="w-2 z-10 flex flex-col self-stretch"
+        style="margin-block: calc(-1 * (var(--spacing-sm) + 1px));"
+      >
+        <div class={"#{route_to_class(@route)} grow top"} />
+        <div class={"#{route_to_class(@route)} grow bottom"} />
+      </div>
+      <.icon
+        id={:erlang.phash2(@arrival)}
+        class={"size-[.75rem] #{route_to_class(@route)} rounded-full ring-2 z-20"}
+        name="circle"
+        style="margin-left: calc(-1.75rem + 2px);"
+        phx-hook="SFRouteRingColor"
+      />
```

Comment:
```suggestion
      <div class={"#{route_to_class(@route)} size-3.5 rounded-full border-xs border-[#00000026] z-20"} />
```

This could be simplified by just using a rounded div and giving it a slightly transparent border (which is how it's represented in the Figma). No need for SVG or JS!